pkg/guestagent/guestagent_linux.go

```diff
@@ -47,15 +47,17 @@ func New(newTicker func() (<-chan time.Time, func()), iptablesIdle time.Duration
 	}
 
 	if auditStatus.Enabled == 0 {
-		if err = auditClient.SetEnabled(true, libaudit.WaitForReply); err != nil {
+		err = auditClient.SetEnabled(true, libaudit.WaitForReply)
+		if err != nil {
```

Comment:
Why?

Reply:
It's the [`sloppyReassign`](https://go-critic.com/overview.html#sloppyreassign) check.

There are two ways of fixing this:

1)
```
if err := auditClient.SetEnabled(true, libaudit.WaitForReply); err != nil {
```

2)
```
err = auditClient.SetEnabled(true, libaudit.WaitForReply); 
if err != nil {
```

I choose the second because it doesn't change semantics of the previous code.

Reply:
The check seems too strict

Reply:
Disabled `sloppyReassign` check and reverted changes.